src/views/List.jsx

```diff
@@ -20,52 +20,58 @@ export function List({ data, listPath }) {
 	});
 
 	function handleInputChange(e) {
+		e.preventDefault();
 		setInput(e.target.value);
 	}
 
 	function clearSearch() {
 		setInput('');
 	}
 
+	const listName = listPath?.substring(listPath.indexOf('/') + 1);
+
+	if (data.data.length === 0) {
+		return (
+			<>
+				<h2>You have no items in this list!</h2>
+				<AddItemForm listPath={listPath} />
+			</>
+		);
+	}
+
 	if (data.loading) {
 		// If data is not loaded yet, render a loading indicator
 		return <p>Loading...</p>;
 	}
 
 	return (
 		<>
-			<form action="">
-				<label htmlFor="searchItems">Search Items: </label>
-				<input
-					onChange={handleInputChange}
-					id="searchItems"
-					value={input}
-					type="text"
-				/>
-			</form>
-			<button onClick={clearSearch}>X</button>
-			<p>
-				Hello from the <code>/list</code> page!
-			</p>
+			<h2>{listName}</h2>
+			<div className="searchInput">
+				<form action="" onSubmit={(e) => e.preventDefault()}>
+					<label htmlFor="searchItems">Search your shopping list: </label>
+					<input
+						onChange={handleInputChange}
+						id="searchItems"
+						value={input}
+						type="text"
+					/>
+				</form>
+				<button onClick={clearSearch}>Clear</button>
+			</div>
+			<div></div>
```

Comment:
Wasn't super clear but meant to wrap divs around content you want to to go to the next line. Also makes it easier to apply a class/style to do spacing.

```javascript
<div>
  {filteredItems.map((item) => (
	<ListItem key={item.id} item={item} listPath={listPath} />
   ))}
</div>
```